fedimint-client/src/oplog.rs

```diff
@@ -291,10 +342,11 @@ fn rev_epoch_ranges(
 ///        functions.
 #[derive(Debug, Serialize, Deserialize)]
 pub struct OperationLogEntry {
-    operation_module_kind: String,
-    meta: serde_json::Value,
+    pub(crate) operation_module_kind: String,
+    pub(crate) meta: serde_json::Value,
     // TODO: probably change all that JSON to Dyn-types
     pub(crate) outcome: Option<serde_json::Value>,
+    pub(crate) outcome_time: Option<SystemTime>,
```

Comment:
nit: Ideally this would be one option with a struct :/

Reply:
done